.github/workflows/e2eForPR.yml

```diff
@@ -23,9 +23,44 @@ jobs:
             if [[ ${changed_file} =~ ^build\.ts$ ]]; then
               cantPassWithoutTests=true
             fi
+            if [[ ${changed_file} =~ src\/mockTests\.ts$ ]]; then
```

Comment:
I believe this should be `scripts/mockTests.ts` instead of `src\/mockTests\.ts` @TheUnderScorer

Reply:
We have to escape the `/` since it's a regex pattern.

Reply:
I wasn't talking about escaping, the path is incorrect. The `mockTests.ts` file is not in `src` folder.

Reply:
Ah! Yep, good catch, fixed!